### PR TITLE
Minor plugin changes: WebSuggest and GoogleTranslate

### PR DESCRIPTION
**Pull request by alexandr-san4ez**
default

Behavior is taken from Launchy (plugin weby).

### CODE REVIEW

File: WebSuggest/websuggest.py [hunk 1]
```diff
@@ -334,6 +334,12 @@ def on_suggest(self, user_input, items_chain):
 
         if suggestions:
             self.set_suggestions(suggestions, kp.Match.ANY, kp.Sort.NONE)
+        else:  # default item if not founded suggestions
```
Comment (polyvertex) [inline WebSuggest/websuggest.py]: Please remove the comment

File: WebSuggest/websuggest.py [hunk 2]
```diff
@@ -334,6 +334,12 @@ def on_suggest(self, user_input, items_chain):
 
         if suggestions:
             self.set_suggestions(suggestions, kp.Match.ANY, kp.Sort.NONE)
+        else:  # default item if not founded suggestions
+            item = current_item.clone()
+            item.set_args(user_input)
+            item.set_short_desc("Suggestions not found (default action: {})".format(
```
Comment (polyvertex) [inline WebSuggest/websuggest.py]: Please replace `Suggestions not found` by `No suggestions found`

**Comment by polyvertex** [comment]
Thanks for that! I had just small remarks (see inline comments) that require minor correction before merging

**Comment by polyvertex** [comment]
Also, do not forget to add yourself to the `Contributors` section of the README file if you wish so (and include PR #3 as well)

**Comment by alexandr-san4ez** [comment]
I look forward to your acceptance of the pull request.

**Comment by alexandr-san4ez** [comment]
I hope you approve of these changes.

**Comment by polyvertex** [comment]
Nice features and thanks for being so involved! Could you please stick to one feature per PR? This is getting a bit messy and it can be cumbersome for people to review your code when only one PR is referenced for several features in the change log, or in case you and I would need to extend the discussion here on a specific feature, which means it would block the others from being merged meanwhile. If you really have hard time grouping your commits that way, at least please stick to one package per PR.

Sorry for the extra work and I hope you understand. I'll add that rule to the Contribute section of the README file so to avoid this situation with other contributors in the future.

**Comment by polyvertex** [comment]
`README.md` updated

**Pull request closed without merging**